doc/api/cli.md

```diff
@@ -1142,9 +1145,9 @@ changes:
                  `--report-uncaught-exception`.
 -->
 
-Enables report to be generated on uncaught exceptions. Useful when inspecting
-the JavaScript stack in conjunction with native stack and other runtime
-environment data.
+Enables report to be generated when the process is exiting for uncaught
+exceptions. Useful when inspecting the JavaScript stack in conjunction with
```

Comment:
nit: I would change to something like: `when the process would exit due to an uncaught exception`

Reply:
Thanks. Updated :)